spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu.rb'
+
+describe Menu_Print do
+    let (:including_class){ Class.new { include Menu_Print } }
+      describe '#show'do
+        test_menu = {food: 1}
+          it 'prints out the menu'do
+              expect { including_class.new.show(test_menu) }.to output("Menu:\nfood. Price: £1.00\n").to_stdout
+            end
+          end
```

Comment:
Lint/BlockAlignment: `end` at 10, 10 is not aligned with `describe '#show'do` at 5, 6.